src/NSwag.SwaggerGeneration.AspNetCore/NSwag.SwaggerGeneration.AspNetCore.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
```

Comment:
Why do we need a nuspec? Isnt this created on compile time for .net standard libs?